src/hawkbit-client.c

```diff
@@ -868,4 +868,3 @@ int hawkbit_start_service_sync()
 
         return res;
 }
-
```

Comment:
Thanks for fixing this. You should squash this commit into your previous one, as the newline was introduced by your previous commit of this PR.